modules/deploy/pages/deployment-option/self-hosted/kubernetes/kubernetes-cluster-requirements.adoc

```diff
@@ -1,70 +1,182 @@
+
+Each Pod replica that runs a Redpanda broker requires its own dedicated worker node for the following reasons:
 
 * Redpanda brokers are designed to have access to all resources, such as CPU and memory, on their host machine.
 * External clients access Redpanda brokers using the address of a worker node. See <<External networking>>.
 
-NOTE: The Helm chart configures `podAntiAffinity` rules to make sure that each Redpanda broker runs on its own worker node.
+NOTE: The Redpanda Helm chart configures `podAntiAffinity` rules to make sure that each Redpanda broker runs on its own worker node.
+
+*Recommendation*: <<Deploy at least three Pod replicas>>.
+
+=== CPU
+
+*Requirement*: Two physical, not virtual, cores for each worker node.
+
+*Requirement*: x86_64 (Westmere or newer) and AWS Graviton family processors are supported.
+
+*Recommendation*: Four physical cores are strongly recommended.
+
+*Recommendation*: <<Set resource requests and limits for memory and CPU>>.
+
+=== Memory
+
+*Requirement*: 2 GB or more of memory per core.
+
+=== Storage
+
+*Requirement*: The Redpanda data directory (`/var/lib/redpanda/data`) and the Tiered Storage cache must be mounted on an XFS or ext4 filesystem. For information about supported volume types for different data, see xref:manage:kubernetes/storage/volume-types.adoc[].
+
+CAUTION: Avoid using NFS (Network File System) for the Redpanda data directory or the Tiered Storage cache.
+
+*Recommendation*: Use an XFS filesystem for its enhanced performance with Redpanda workloads.
+
+*Recommendation*: <<kubernetes-volumes, Use local PersistentVolumes that are backed by locally attached NVMe disks>> to store the Redpanda data directory.
+
+*Recommendation*: For setups with multiple disks, use a RAID-0 (striped) array. It boosts speed but lacks redundancy. A disk failure can lead to data loss.
+
+=== Cloud instance types
+
+*Recommendation*: Use instance types that supports local NVMe disks. See xref:./cloud-instance-local-storage.adoc[].
+
+For help sizing your Kubernetes cluster, see xref:deploy:deployment-option/self-hosted/manual/sizing.adoc[Sizing Guidelines].
+
+=== Security
+
+*Recommendation*: If you're using a cloud platform, use xref:manage:security/iam-roles.adoc[IAM roles] to restrict access to resources in your cluster.
+
+=== Object storage providers for Tiered Storage
+
+Redpanda supports the following storage providers for Tiered Storage:
+
+- Amazon Simple Storage Service (S3)
+- Google Cloud Storage (GCS), using the Google Cloud Platform S3 API
+- Azure Blob Storage (ABS)
+
+=== External networking
+
+*Requirement*: Minimum 10 GigE (10 Gigabit Ethernet). A 10 GigE connection ensures:
+
+* High data throughput
+* Reduced data transfer latency
+* Scalability for increased network traffic
 
-== Storage volumes
+*Requirement*: For external access, each worker node in your cluster must have a static, externally accessible IP address.
```

Comment:
Curious to know why the option to place the selected text [+/-] no longer appears to be available in this PR?

*Requirement*: For external access, each worker node in your cluster must have a static, externally-accessible IP address.

Reply:
I think you were working on an outdated version of the file

Reply:
I was following the advice here: https://www.grammarly.com/blog/hyphen/#:~:text=You%20never%20use%20a%20hyphen%20when%20your%20modifier%20is%20made%20up%20of%20an%20adverb%20ending%20in%20%2Dly%20and%20an%20adjective.

> You never use a hyphen when your modifier is made up of an adverb ending in -ly and an adjective.

If our advice is different, we should add it to our style guide.